client/my-sites/site-indicator/site-indicator.jsx

```diff
@@ -248,11 +253,15 @@ class SiteIndicator extends Component {
 			accessFailedMessage = (
 				<span>
 					{ translate( 'This site cannot be accessed.' ) }
-					<DisconnectJetpackButton
```

Comment:
Is this component only used here? If so, should we remove it altogether?

Reply:
Not really. It's also used in https://github.com/Automattic/wp-calypso/blob/5f15f3365213253724ca0d8798c6cb8160f00bc9/client/my-sites/plugins/plugin-site-disabled-manage/index.jsx so let's leave it for now.